### ISSUE TITLE
Inappropriate bar labels.

### ISSUE THREAD

**Issue by 1hef001** [verigak/progress#81]
Am I doing something wrong here to get "?[K" added to the necessary string?

code :
```
bar = IncrementalBar('Cleaning text', max=len(links), suffix='%(percent)d%%' )
bar = IncrementalBar('Scraping Links', max=len(paper.articles), suffix='%(percent)d%%')
```

Output:

```
?[KScraping Links |████████████████████████████████| 100%
getting content
?[KCleaning text |████████████████████████████████| 100%
scraper_main
?[KScraping Links |████████████████████████████████| 100%
getting content
?[KCleaning text |████████████████████████████████| 100%
scraper_main
?[KScraping Links |████████████████████████████████| 100%
```

**Comment by zokai** [comment]
I have the same problem for the Bar.
Also Spinner splits the several letters from message and repeats after the spinner.
Scanning subfolders... -s...

**Comment by zhaowb** [comment]
That's 'clearln()' sent esc code to clear line.
There is no way to disable that behaviour but the design provided a more generic way to bypass:
```
from progress.bar import Bar
bar = Bar(clearln=lambda: None)
bar.next()
```
Here Bar is sample for any other classes. The constructore is to demostrate how to set a dummy clearln function, other parameters are still acceptable.

**Comment by verigak** [comment]
This issue should be fixed with 12e46ed70209bdc3bb8ca1f0416039883b94e4e1
